Flixter/DetailsViewController.m

```diff
@@ -54,14 +55,13 @@ - (void)viewDidLoad {
     [self.movieSynopsisLabel sizeToFit];
 }
 
-/*
 #pragma mark - Navigation
 
 // In a storyboard-based application, you will often want to do a little preparation before navigation
 - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
-    // Get the new view controller using [segue destinationViewController].
-    // Pass the selected object to the new view controller.
+    NSString *movieIdString = [NSString stringWithFormat:@"%@", self.movieDetails[@"id"]];
```

Comment:
This has term "Hungarian Notation" and is almost always a bad idea when naming things.

The reason is that you want this variable to be the movie id. If the type changes to an int or a more qualified type of string, you may forget to rename this and now the code is confusing to read.